src/gmt_common_longoptions.h

```diff
@@ -107,21 +107,26 @@
     {   0, 'p', "perspective",
                 "x,y,z",                 "x,y,z",
                 "v,w",                   "view,world" },
-    { ',', 'q', "inrows",
+    { ',', 'q', "inrows",                /* also note special -qi code in gmtinit_translate_to_short_options()!! */
                 "~",                     "invert",
-                "a,c,f,s",               "byset,column,byfile,bysegment" },    /* Actually -qi */
-    { ',', 'q', "outrows",
+                "a,c,t,s",               "byset,column,bytable,bysegment" },
+    { ',', 'q', "outrows",               /* also note special -qo code in gmtinit_translate_to_short_options()!! */
                 "~",                     "invert",
-                "a,c,f,s",               "byset,column,byfile,bysegment" },    /* Actually -qo */
+                "a,c,t,s",               "byset,column,bytable,bysegment" },
     {   0, 'r', "registration",
                 "g,p",                   "gridline,pixel",
                 "",                      "" },
     {   0, 's', "skiprows",
                 "",                      "",
                 "a,r",                   "any,reverse" },
-    {   0, 't', "transparency",          "", "", "", "" },
+    {   0, 't', "transpercent",
```

Comment:
``transpercent``? Firs time I see this word. I don't think it's better then ``transparency`` or ``alpha``.

Reply:
I missed that.  Let's stick with transparency here, @rbdavis.

Reply:
Paul originally used 'transparency' for the -t common option. This was fine by itself, but now working in grdimage I see there is a -Q option that also sets a transparency parameter. These need to be distinguished, I think, so I (i) changed the -t to 'transpercent' to indicate that it specifies a transparency percentage, and (ii) used 'transnode' for grdimage -Q to indicate that it controls per-node transparency. I think that 'transparency' is probably too general a term to be used in either case.

Reply:
I think it is better to change what -Q means rather than -t.  -Q does related to transparency in the sense nodes that are NaN will be see-through.  But it is therefore on/off, not like transparency which can be partial.  So maybe --nanmask or just --mask is better here.

Reply:
OK.